QuickCheck.cabal

```diff
@@ -254,7 +254,7 @@ Test-Suite test-quickcheck-misc
     Default-language: Haskell2010
     hs-source-dirs: tests
     main-is: Misc.hs
-    build-depends: base, QuickCheck
+    build-depends: base, QuickCheck, containers, nothunks
```

Comment:
I'd say `nothunks` is a bad idea, as it doesn't support all the GHCs `QuickCheck` does.

Reply:
flag for tests is a ugly. Users don't care about it, but they see it. Also you'll forget to toggle it.

Having a module which is conditional on `if impl(ghc >= ...)` is better.

Reply:
yes, derp. that's what I meant.